### PR TITLE
AC-652 : Need for better Play Store listing for Android Client

### PR DESCRIPTION
**Pull request by VibhorChinda**
## Description of what I changed

I have added an "about us" text view in login fragment and linked that text view to the OpenMRS android user guide.
Link -> https://openmrs.github.io/openmrs-android-client-user-guide/getting-started.html

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-652

## Checklist: I completed these to help reviewers :)

- [X] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [X] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [X] All new and existing **tests passed**.

- [X] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by VibhorChinda** [comment]
WhatsApp Image 2020-01-13 at 5 32 51 PM

**Comment by VibhorChinda** [comment]
@f4ww4z sir please have a look at the PR.

**Comment by ribhavsharma** [comment]
Hey vibhor! How about making the text the same color and size as the ```Unable to Login?``` part?

**Comment by VibhorChinda** [comment]
> Hey vibhor! How about making the text the same color and size as the `Unable to Login?` part?

@ribhavsharma tried to do as you requested. But did not look good. As i guess it should look different on the login page.

**Comment by tbagz104** [comment]
@f4ww4z @VibhorChinda  I have created an alternative approach for the about page which is much more convenient [https://issues.openmrs.org/browse/AC-657](https://issues.openmrs.org/browse/AC-657) Could you please take a look at this?

File: openmrs-client/src/main/res/layout/fragment_login.xml [hunk 1]
```diff
@@ -59,7 +59,7 @@
                 android:layout_marginBottom="10dp"
                 android:gravity="center"
                 android:orientation="vertical"
-                android:visibility="gone">
+                android:visibility="visible">
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/layout/fragment_login.xml]: Why this change?

File: openmrs-client/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -417,6 +417,7 @@
     <string name="intro_manage_desc">You can manage providers using our app.</string>
     <string name="intro_location_desc">You can change your location at a simple click.</string>
     <string name="intro_settings_desc">Try out our dark mode feature.\nExciting? Let\'s get started.</string>
+    <string name="about_us">About Us</string>
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/values/strings.xml]: Change to `Get started with our user guide`, and rename the string.

File: openmrs-client/src/main/res/layout/fragment_login.xml (hunk 1 above)
Comment (VibhorChinda) [reply openmrs-client/src/main/res/layout/fragment_login.xml]: > Why this change?

@f4ww4z basically if anyone needs to change the UI of login fragment, then they have to make the views visible first. I thought there is no harm in keeping the views in visible state only by default. So i made this change.

**Comment by VibhorChinda** [comment]
WhatsApp Image 2020-01-16 at 8 57 31 AM

**Comment by VibhorChinda** [comment]
@f4ww4z please have a look sir.
